packages/pf4-component-mapper/src/files/wizard.js

```diff
@@ -157,7 +159,8 @@ WizardInternal.propTypes = {
   closeButtonAriaLabel: PropTypes.string,
   hasNoBodyPadding: PropTypes.bool,
   navAriaLabel: PropTypes.string,
-  container: PropTypes.instanceOf(Element)
+  container: PropTypes.instanceOf(Element),
+  StepTemplate: PropTypes.func
```

Comment:
What if somebody provides a class component? I would use the `componentType`

Reply:
Don't you think elementType?

Reply:
yeah